args_parser.go

```diff
@@ -108,6 +109,10 @@ func newKingpinParser() argsParser {
 			" chain and host name").
 		Short('k').
 		BoolVar(&kparser.insecure)
+	app.Flag("disableKeepAlives",
+		"Disable HTTP KeepAlive").
```

Comment:
I think it should also mention that for `fasthttp` a header needs to be set. [See here.](https://github.com/valyala/fasthttp/issues/623#issuecomment-520124446)
Something like "Disable HTTP keep-alive. For fasthttp also set -H 'Connection: close'".